Traceback (most recent call last):
  File /home/fud3/Documents/git/3y3/./dados.py, line 39, in <module>
    relatorio.write_performance(current_time, cpu, ram, disk)
TypeError: write_performance() takes 3 positional arguments but 4 were given

